engine/src/main/scala/cromwell/engine/workflow/workflowstore/WorkflowStoreActor.scala

```diff
@@ -144,11 +145,15 @@ case class WorkflowStoreActor(store: WorkflowStore, serviceRegistryActor: ActorR
   /**
     * Takes the workflow id and sends it over to the metadata service w/ default empty values for inputs/outputs
     */
-  private def registerIdWithMetadataService(id: WorkflowId): Unit = {
+  private def registerSubmissionWithMetadataService(id: WorkflowId, sourceFiles: WorkflowSourceFiles): Unit = {
     val submissionEvents = List(
       MetadataEvent(MetadataKey(id, None, WorkflowMetadataKeys.SubmissionTime), MetadataValue(OffsetDateTime.now.toString)),
       MetadataEvent.empty(MetadataKey(id, None, WorkflowMetadataKeys.Inputs)),
-      MetadataEvent.empty(MetadataKey(id, None, WorkflowMetadataKeys.Outputs))
+      MetadataEvent.empty(MetadataKey(id, None, WorkflowMetadataKeys.Outputs)),
+
+      MetadataEvent(MetadataKey(id, None, WorkflowMetadataKeys.SubmissionSection, WorkflowMetadataKeys.SubmissionSection_Workflow), MetadataValue(sourceFiles.wdlSource)),
```

Comment:
Considering how similar these all are, having some function representing the common parts would cut down on the visual chatter - although I can't say that I really care all that much here